### ISSUE TITLE
Generate components with zero paths in the service

### ISSUE THREAD

**Issue by agaro1121** [smithy-lang/smithy#2360]
Is there a way to generate OpenAPI components/schemas that are not used in a `path`?

I tried converting a model and I only get schemas when they are used in a path. I set `config.setKeepUnusedComponents(true)` but that may not be doing what I think it does.

**Comment by JordonPhillips** [comment]
Do you mean shapes that aren't connected to the service through an operation? If so, no.

**Comment by agaro1121** [comment]
Yeah that's what I meant. I eventually found the code path that has the constraint.
